src/prometheus_alert_rules/nvme.rule

```diff
@@ -0,0 +1,32 @@
+---
+groups:
+  - name: nvme
+    rules:
+      - record: available_file_size_percentage
+        expr: |
+          100 *  (node_filesystem_avail_bytes / node_filesystem_size_bytes)
```

Comment:
Are you going to use these in dashboards? If not, it's not worth the overhead to create a separate calculated timeseries with a recording rule.

Reply:
I saw there is one similar in dashboard already. I will remove it here.